content/workshops/pki_workshop/unit_lessons/unit_4.md

```diff
@@ -0,0 +1,87 @@
+# Unit 4: Accessing the Web Interface
+
+In this unit you will access the web interface of each subsystem. Each Certificate System instance is a Tomcat server instance and each Certificate System subsystem (such as CA or KRA) is deployed as a web application in Tomcat.
+
+All subsystems share a common administrative interface and have an agent interface that allows for agents to perform the tasks assigned to them. A CA Subsystem has an end-entity interface that allows end-entities to enroll in the PKI. An OCSP Responder subsystem has an end-entity interface allowing end-entities and applications to check for current certificate revocation status. Finally, a TPS has an operator interface.
+
```

Comment:
All subsystems share a common administrative interface and have an agent interface that [enables] agents to perform the tasks assigned to them. A CA [subsystem] has an end-entity interface that [enables] end-entities to enroll in the PKI. An OCSP Responder subsystem has an end-entity interface, [which enables] end-entities and applications to check for current certificate revocation status. Finally, a TPS has an operator interface.